app/uk/gov/hmrc/individualsmatchingapi/controllers/CommonController.scala

```diff
@@ -42,6 +43,8 @@ import scala.util.{Failure, Success, Try}
 
 abstract class CommonController @Inject()(cc: ControllerComponents) extends BackendController(cc) {
 
+  val logger: Logger = Logger(getClass)
```

Comment:
can we use the Logging mixin instead? `with Logging`

Reply:
We should indeed, getClass is for pre- Play-2.8, or was it pre- Play-2.7?